<?xml version='1.0'?>
<ns2:test-suite xmlns:ns2='urn:model.allure.qatools.yandex.ru' start='1689078936979' stop='1689078939140'>
    <name>Faker:</name>
    <title>Faker:</title>
    <test-cases>
        <test-case start='1689078938096' status='failed' stop='1689078938799'>
            <name>FakerDemo</name>
            <title>FakerDemo</title>
            <labels>
                <label name='language' value='javascript'/>
                <label name='framework' value='codeceptjs'/>
            </labels>
            <parameters/>
            <steps/>
            <attachments>
                <attachment title='Main session - Last Seen Screenshot' source='17125cdd-0cfe-4b6b-b4b3-56980235e637-attachment.png' type='image/png' size='6897'/>
            </attachments>
            <failure>
                <message>name is not defined</message>
                <stack-trace>ReferenceError: name is not defined
                    at Test.&lt;anonymous> (C:\POC\tests\faker.js:11:15)
                    at test.fn (C:\POC\node_modules\codeceptjs\lib\scenario.js:92:14)
                    at callFnAsync (C:\POC\node_modules\mocha\lib\runnable.js:394:21)
                    at Runnable.run (C:\POC\node_modules\mocha\lib\runnable.js:338:7)
                    at Runner.runTest (C:\POC\node_modules\mocha\lib\runner.js:666:10)
                    at C:\POC\node_modules\mocha\lib\runner.js:789:12
                    at next (C:\POC\node_modules\mocha\lib\runner.js:581:14)
                    at C:\POC\node_modules\mocha\lib\runner.js:591:7
                    at next (C:\POC\node_modules\mocha\lib\runner.js:474:14)
                    at cbHookRun (C:\POC\node_modules\mocha\lib\runner.js:539:7)</stack-trace>
            </failure>
        </test-case>
    </test-cases>
</ns2:test-suite>